source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
 Configuration
 -------------
 
-Minimal "sane" config:
+The configuration of Unbound can be a little tricky due to the extensive array of configurable options. Below we will go through a basic, recommended config, but feel free to add and experiment with options as you need them. Also feel free to remove lines that are commented out (uncommenting bu removing the "#") if they are not nesecary in your setup.
```

Comment:
Saying it is a little tricky could deter users from using Unbound. Is it really tricky? Doesn't Unbound come with sane defaults out-of-the-box?

Reply:
Suggestion:
"Unbound has a vast array of configuration options for advanced use cases, which can seem a little overwhelming at first. Luckily, all of the defaults are sensible and secure, so in a lot of environments you can run Unbound without changing any options. Below we will go through a basic, recommended config, but feel free to add and experiment with options as you need them..."